source/renderer/app/stores/ada/AdaRedemptionStore.js

```diff
@@ -62,6 +62,7 @@ export default class AdaRedemptionStore extends Store {
     ipcRenderer.on(PARSE_REDEMPTION_CODE.ERROR, this._onParseError);
     this.registerReactions([
       this._resetRedemptionFormValuesOnAdaRedemptionPageLoad,
+      this._redirectToAddWalletBeforeRedemption,
```

Comment:
@DominikGuzei I don't think we need this redirect anymore as handle this case by showing special content on the Ada redemption screen.